ManatiCR/manatime
# 62: Timesheet daily view.

**Steps to test**
- Add a timesheet node with the starter date of the monday of the current week and the finish date of the sunday of the curret week.
- Add some time entry nodes with different dates between the current week.
- Go to `timesheet/daily` and see if you can select the time entries by chosen the week's day.

[Issue link](https://github.com/ManatiCR/manatime/issues/62)
